src/services/autoLauncher.ts

```diff
@@ -1,12 +1,12 @@
+import { isPlatform } from '../utils/main';
 import { AutoLauncherClient } from '../types/autoLauncher';
-import { isPlatformWindows, isPlatformMac } from '../utils/main';
 import linuxAutoLauncher from './autoLauncherClients/linuxAutoLauncher';
 import macAndWindowsAutoLauncher from './autoLauncherClients/macAndWindowsAutoLauncher';
 
 class AutoLauncher {
     private client: AutoLauncherClient;
     init() {
-        if (isPlatformMac() || isPlatformWindows()) {
+        if (isPlatform('mac') || isPlatform('windows')) {
```

Comment:
This is susceptible to typos? We should either use enums or use verbose functions (like before).

Reply:
>export function isPlatform(platform: 'mac' | 'windows' | 'linux') {

https://github.com/ente-io/bhari-frame/pull/113/files#diff-9ec358ff9cd91db2ba4e07746164e5ee67b7af2093340c308fa7174f029e654aR87 

the isPlatform function only accepts ->  'mac' , 'windows' , 'linux' as parameter , so there is no chance of typos